ai-machine-learning/thoth-thamos/intro.md

```diff
@@ -0,0 +1,10 @@
+In this tutorial, we are going to run Thamos Advise on a simple Python app using Thamos CLI. 
```

Comment:
In this tutorial, we are going to run Thamos Advise on a simple TensorFlow Python app using Thamos CLI.